src/utils/helpers/oracle.ts

```diff
@@ -0,0 +1,22 @@
+import { CurrencyExt } from '@interlay/interbtc-api';
+import { Bitcoin, Currency, ExchangeRate } from '@interlay/monetary-js';
+
+// This is for simulating when the oracle is down.
+// import { GOVERNANCE_TOKEN_SYMBOL, RELAY_CHAIN_NATIVE_TOKEN_SYMBOL } from '@/config/relay-chains';
+
+const getExchangeRate = async (
```

Comment:
I created this abstraction since we have some potential error cases in `VaultsTable` (src/pages/Dashboard/sub-pages/Vaults/VaultsTable/index.tsx) that we may want to address further.

Currently regardless of any exchange rate (governance token exchange rate or relay-chain native token exchange rate) is broken (from the oracle), all of the vaults are replaced with an error message by the relevant error boundary.
We could make it that only the vaults with the unavailable exchange rate are replaced with error messages.

Reply:
Hi Anton, nice solution :) longer term the fix for this will possibly be in the lib (i.e. returning error status codes, or warnings) but it's great having this here as that's not something we'll be working on any time soon.